src/main/resources/static/js/common.js

```diff
@@ -308,6 +308,16 @@ function executeIfLdapIsDeactivated() {
     $("#ldapUserName").text("");
 }
 
+function downloadSubscriptions(subscriptionData) {
+    var foundSubscriptions = subscriptionData.foundSubscriptions;
+    var jsonData = JSON.stringify(foundSubscriptions, null, 2);
+    if (jsonData == undefined) {
+        logMessage("Subscription data to download was not found");
+        return;
+    }
+    downloadFile(jsonData, "application/json;charset=utf-8", "subscriptionsData.json");
```

Comment:
I'm thinking the name of the file can be the subscription name when downloading a single subscription. Makes it easier to find :) Either way the name of the file should be consistent plural or singular depending on if you download only one subscription or several using the bulk download.

Reply:
Done.